tutorials/examples/cov2d_simulation.py

```diff
@@ -61,9 +61,8 @@
 logger.info(f'Load 3D map and downsample 3D map to desired grids '
             f'of {img_size} x {img_size} x {img_size}.')
 infile = mrcfile.open(os.path.join(DATA_DIR, 'clean70SRibosome_vol_65p.mrc'))
-vols = infile.data
-vols = vols[..., np.newaxis]
-vols = downsample(vols, (img_size*np.ones(3, dtype=int)))
+vols = Volume(infile.data)
+vols = vols.downsample((img_size*np.ones(3, dtype=int)))
```

Comment:
Agreed. I just changed Volume.downsample to admit an integer. It can construct the tuple and send to downsample for us. This just becomes ` vols = vols.downsample(img_size)` then.